config-controller/test/e2e/e2e_test.go

```diff
@@ -0,0 +1,121 @@
+/*
+Copyright 2024.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package e2e
+
+import (
+	"fmt"
+	"os/exec"
+	"time"
+
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+	"github.com/stackrox/rox/config-controller/test/utils"
+)
+
+const namespace = "config-controller-system" // #nosec G101
```

Comment:
Is this auto-generated? I think we need to review the auto-generated stuff and use kubebuilder. This feels out of touch with what our config does.

Reply:
This file is generated by both operator-sdk and kubebuilder. I can remove it if we don't think we'll ever use it.  I mean I could remove it even if we do think we'll use it since it's so easy to regenerate.

Reply:
I think it can stay in that case. My main concern was the generation of code from operator-sdk we do not need.